Jenkinsfile

```diff
@@ -50,13 +40,25 @@ library(
   retriever: modernSCM([$class: 'GitSCMSource',remote: "${params.LIBRARIES_REPO}"])
 )
 
-List arches = params.ARCHES.tokenize(',')
 def errorMessages = ''
 def config = MAQEAPI.v1.getProvisioningConfig(this)
+config.jobgroup = 'multiarch-qe'
+
+def targetHost = MAQEAPI.vi.newTargetHost()
+targetHost.hostname = params.PREPROVISIONED_HOST
+targetHost.name = params.PREPROVISIONED_HOST
+targetHost.arch = 'x86_64'
+targetHost.provisioner = 'NOOP'
+
+withCredentials([usernamePassword(credentialsId:'osp-jenkins-private-key', 
```

Comment:
osp-jenkins-private-key is a "secretFile" not a usernamePassword.

You need to use the same credentialId from when we set up the username password creds for the gitlab account.
```
withCredentials([
        usernamePassword(credentialsId:"osp16-gitlab-consulting-rh",
                         usernameVariable:'USERNAME',    
                         passwordVariable:'PASSWORD'),
        ]){
                        targetHost.scriptParams = "$USERNAME $PASSWORD"
        }
```